src/udiskslinuxmanager.c

```diff
@@ -530,6 +530,27 @@ wait_for_array_object (UDisksDaemon *daemon,
   return ret;
 }
 
+static gchar* md_node_from_name (const gchar *name, GError **error) {
```

Comment:
Please add a comment making clear that this should be replaced by a call of the *libblockdev*'s ``md_node_from_name()`` function. So that we don't forget to do so.